pyscriptjs/src/main.ts

```diff
@@ -203,6 +203,21 @@ export class PyScriptApp {
         // XXX: maybe the following calls could be parallelized, instead of
         // await()ing immediately. For now I'm using await to be 100%
         // compatible with the old behavior.
+
+        // needed for patch inside pyscript.py
+        await runtime.installPackage("matplotlib");
```

Comment:
I'm slightly concerned about installing matplotlib all the time even if we don't need it. 

What do you think about adding the patch into a try/pass instead?